### PR TITLE
Add ons

### PR DESCRIPTION
**Pull request by eroiyo**
This is a boilerplate for futures React Projects!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,13 @@
...
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
-
-## Test/Deployment Actions
-
-Feel free to add your own deployment actions which can run your tests and deploy to Heroku.
-
-Make sure that you do not modify the [`.github/workflows/linters.yml`](.github/workflows/linters.yml) but that you create a separe GitHub Action workflow file for that.
+-React
+-Javascript
+-HTML
```
Comment (vichuge) [inline README.md:13]: - [ ] Please, add more info on the readme file to make more professional. You could follow the [microverse template](https://github.com/microverseinc/readme-template) to know more about it. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.
- [ ] After I install the project locally and try to run it, this error appears, I kindly invite you to work on it. 🐶 
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (eroiyo) [reply README.md:13]: Hello, thanks for the review!

can you tell me how you do to run my project in the local version?

Comment (eroiyo) [reply README.md:13]: oh i see, thanks

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @eroiyo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks!

**Pull request merged**